src/plugins/vis_type_timeseries/public/application/components/add_delete_buttons.tsx

```diff
@@ -17,13 +17,29 @@
  * under the License.
  */
 
-import PropTypes from 'prop-types';
-import React from 'react';
-import { EuiToolTip, EuiButtonIcon, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import React, { MouseEvent } from 'react';
+import { EuiButtonIcon, EuiFlexGroup, EuiFlexItem, EuiToolTip } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { isBoolean } from 'lodash';
 
-export function AddDeleteButtons(props) {
+interface AddDeleteButtonsProps {
+  addTooltip: string;
+  deleteTooltip: string;
+  cloneTooltip: string;
+  activatePanelTooltip: string;
+  deactivatePanelTooltip: string;
+  isPanelActive?: boolean;
+  disableAdd?: boolean;
+  disableDelete: boolean;
+  responsive?: boolean;
+  testSubj: string;
+  togglePanelActivation?: () => void;
+  onClone?: () => void;
+  onAdd: () => void;
+  onDelete: (event: MouseEvent) => void;
+}
+
+export function AddDeleteButtons(props: AddDeleteButtonsProps) {
```

Comment:
Does it make sense to type in addition the test file for this component: `add_delete_buttons.test.js` ?